modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -4,11 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, examples
 
-
-The examples demonstrate common data extraction and transformation approaches.
+The following DataWeave examples demonstrate common data extraction and transformation approaches. There are general examples of how to transform data, and also examples of how to use DataWeave in Mule runtime engine (Mule) applications.
```

Comment:
```suggestion
The following DataWeave examples demonstrate common data extraction and transformation approaches. The examples illustrate transformations in DataWeave scripts and in Mule configuration XML.
```

Reply:
Not sure if my suggestion captures the division precisely. Others might have better suggestions.

Reply:
Maybe something like: 

`The following DataWeave examples demonstrate common data extraction and transformation approaches. There are DataWeave code examples of how to transform data, and also examples of Mule applications that implemente DataWeave transformations.`

Reply:
From having reviewed this conversation, at this point I suggest that we take the exact wording that you decide upon for the intro text and convert it into two parallel subtitles. It is important to:

- Use introductory text that implies the distinction between the two subtopics.

- Create brief but parallel subtitles that represent the distinct nature of each subtopic: at this point, it appears that "transform(nation)" is the term that the two subtopics have in common:
     - DataWeave Data Transformation Code
     - DataWeave Data Transformation (Code) as Implemented by Mule Applications

- Text that immediately follows each subtitle and expands upon it to make it perfectly clear (title+short description=full explanation of the content within each subtopic)
     - The following examples focus on application-agnostic DataWeave data extraction and transformation....
     - The following examples focus on DataWeave data extraction and transformation as implemented in Mule applications....
     
The exact titles and short descriptions will depend on what wording you choose to capture the division in a way that your audience will understand.